src/go/rpk/pkg/cli/args_test.go

```diff
@@ -1,80 +0,0 @@
-// Copyright 2020 Vectorized, Inc.
```

Comment:
If this file and src/go/rpk/pkg/cli/args.go are introduced to be deleted, I think it's simpler to just remove the first two commits.

Reply:
Alternatively, you could squash the 1st, 2nd and 3rd (this) commit.